.github/ISSUE_TEMPLATE.md

```diff
@@ -5,8 +5,8 @@
 
 ##### Reproduction Steps
 1. [Step 1]
-1. [Step 2]
-1. [Step 3]
+2. [Step 2]
```

Comment:
No, I'd prefer to leave it as is. For those who do use it, such as myself, it's a big timesaver. We'd switch from you changing it to 1.2.3. to me changing it to 1.1.1. Since the 1x3 method saves time and is preferred, I'd rather leave it as is.